pyblish_bumpybox/plugins/validate_scene_version.py

```diff
@@ -6,8 +8,12 @@ class ValidateSceneVersion(pyblish.api.ContextPlugin):
 
     order = pyblish.api.ValidatorOrder
     label = "Scene Version"
+    optional = True
 
     def process(self, context):
 
-        msg = "Could not find a version number in the scene name."
-        assert context.has_data("version"), msg
+        name, ext = os.path.splitext(context.data("currentFile"))
+        msg = "Could not find a version number in the scene name. Please add "
+        msg += "v[number] to the scene name, for example: "
+        msg += "\"{0}_v001{1}\".".format(name, ext)
+        assert "version" in context.data, msg
```

Comment:
> Ah, yeah, think I'd completely lost the plot on what was going on here. Would be better to use a variable name indicative of the purpose of the variable - for instance failure_message or something.

Sure, we could do that.

> Additionally, may be better to break the string over multiple lines, rather than building the string up

One issue is how pyblish_lite handles multi lines, which is not great. I see why we don't want to accommodate for the different GUI's behaviors.

> and use single quotes for the start/end of string to save having to escape the double quotes within.

Now that will be a question of the style we want to stick to. I've been using double quotes throughout for consistency, and it aligns with pyblish's coding style. I would prefer to stick with one style throughout.